docs/pages/api-docs/x-grid.md

```diff
@@ -10,94 +10,93 @@ import { XGrid } from '@material-ui/x-grid';
+|                                     | <span class="prop-name">.MuiDataGrid-colCellCenter</span>     | Styles applied to the centered header cell element.                 |
+|                                     | <span class="prop-name">.MuiDataGrid-colCellRight</span>      | Styles applied to the aligned right header cell element.            |
+|                                     | <span class="prop-name">.MuiDataGrid-colCellTitle</span>      | Styles applied to the header cell title element.                    |
+|                                     | <span class="prop-name">.MuiDataGrid-columnSeparator</span>   | Styles applied to the header cell separator element.                |
+|                                     | <span class="prop-name">.MuiDataGrid-iconSeparator</span>     | Styles applied to the header cell separator icon element.           |
+|                                     | <span class="prop-name">.MuiDataGrid-dataContainer</span>     | Styles applied to the data container element.                       |
+|                                     | <span class="prop-name">.MuiDataGrid-window</span>            | Styles applied to the window element.                               |
+|                                     | <span class="prop-name">.MuiDataGrid-viewport</span>          | Styles applied to the viewport element.                             |
+|                                     | <span class="prop-name">.MuiDataGrid-row</span>               | Styles applied to the row element.                                  |
+|                                     | <span class="prop-name">.Mui-selected</span>                  | Styles applied to the selected row element.                         |
+|                                     | <span class="prop-name">.MuiDataGrid-cellWithRenderer</span>  | Styles applied to the customised cell element.                      |
+|                                     | <span class="prop-name">.MuiDataGrid-withBorder</span>        | Styles applied to the cell element that has right border displayed. |
+|                                     | <span class="prop-name">.MuiDataGrid-cellRight</span>         | Styles applied to the aligned right cell element.                   |
+|                                     | <span class="prop-name">.MuiDataGrid-cellCenter</span>        | Styles applied to the centered cell element.                        |
+|                                     | <span class="prop-name">.MuiDataGrid-footer</span>            | Styles applied to the footer element.                               |
+|                                     | <span class="prop-name">.MuiDataGrid-rowCount</span>          | Styles applied to the footer row count element.                     |
+|                                     | <span class="prop-name">.MuiDataGrid-selectedRowCount</span>  | Styles applied to the footer selected row count element.            |
```

Comment:
yarn prettier

Reply:
Thanks, I did run prettier but idk why it didn't format this